stubs/fpdf2/fpdf/actions.pyi

```diff
@@ -1,26 +1,29 @@
+from _typeshed import Incomplete
 from abc import ABC
-from typing import Any
+
+from .syntax import PDFObject
 
 class Action(ABC):
-    def dict_as_string(self) -> None: ...
+    def __init__(self, next_action: PDFObject | str | None = ...) -> None: ...
```

Comment:
```suggestion
    next: PDFObject | str | None
    def __init__(self, next_action: PDFObject | str | None = ...) -> None: ...
```

Reply:
Ahh, sorry, I thought I'd got that, but I think Github didn't apply the suggestion because of a merge conflict.